### PR TITLE
DOCSUP-14092: Documented the settings for inserting nested

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #25902

### CODE REVIEW

File: docs/en/interfaces/formats.md [hunk 1]
```diff
@@ -1328,6 +1330,8 @@ You can insert Arrow data from a file into ClickHouse table by the following com
 $ cat filename.arrow | clickhouse-client --query="INSERT INTO some_table FORMAT Arrow"
 ```
 
+To insert `Array(Struct)` columns into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns use the setting [input_format_arrow_import_nested](../operations/settings/settings.md#input_format_arrow_import_nested).
```
Comment (olgarev) [inline docs/en/interfaces/formats.md:1333]:
```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
```

Comment (olgarev) [inline docs/en/interfaces/formats.md:1333]:
```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_arrow_import_nested](../operations/settings/settings.md#input_format_arrow_import_nested) setting.
```

File: docs/en/interfaces/formats.md [hunk 2]
```diff
@@ -1384,6 +1388,8 @@ You can insert ORC data from a file into ClickHouse table by the following comma
 $ cat filename.orc | clickhouse-client --query="INSERT INTO some_table FORMAT ORC"
 ```
 
+To insert `Array(Struct)` columns into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns use the setting [input_format_orc_import_nested](../operations/settings/settings.md#input_format_orc_import_nested).
```
Comment (olgarev) [inline docs/en/interfaces/formats.md:1391]:
```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_orc_import_nested](../operations/settings/settings.md#input_format_orc_import_nested) setting.
```

File: docs/en/sql-reference/data-types/nested-data-structures/nested.md [hunk 3]
```diff
@@ -101,4 +103,8 @@ For a DESCRIBE query, the columns in a nested data structure are listed separate
 
 The ALTER query for elements in a nested data structure has limitations.
 
+## Inserting {#inserting}
+
+It is possible to insert `Array(Struct)` columns into nested columns with the settings [input_format_parquet_import_nested](../../../operations/settings/settings.md#input_format_parquet_import_nested), [input_format_arrow_import_nested](../../../operations/settings/settings.md#input_format_arrow_import_nested) and [input_format_orc_import_nested](../../../operations/settings/settings.md#input_format_orc_import_nested) in [Parquet](../../../interfaces/formats.md#data-format-parquet), [Arrow](../../../interfaces/formats.md#data_types-matching-arrow) and [ORC](../../../interfaces/formats.md#data-format-orc) input formats correspondingly.
```
Comment (olgarev) [inline docs/en/sql-reference/data-types/nested-data-structures/nested.md:108]:
```suggestion
To insert `Array(Struct)` values into `Nested` columns, you must switch on the [input_format_parquet_import_nested](../../../operations/settings/settings.md#input_format_parquet_import_nested), [input_format_arrow_import_nested](../../../operations/settings/settings.md#input_format_arrow_import_nested) or [input_format_orc_import_nested](../../../operations/settings/settings.md#input_format_orc_import_nested) settings for [Parquet](../../../interfaces/formats.md#data-format-parquet), [Arrow](../../../interfaces/formats.md#data_types-matching-arrow) and [ORC](../../../interfaces/formats.md#data-format-orc) input formats respectively.
```

**Comment by gyuton** [comment]
@Avogar Hi! Can you review this, please?

File: docs/en/interfaces/formats.md [hunk 4]
```diff
@@ -1270,6 +1270,8 @@ You can insert Parquet data from a file into ClickHouse table by the following c
 $ cat {filename} | clickhouse-client --query="INSERT INTO {some_table} FORMAT Parquet"
 ```
 
+To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
```
Comment (Avogar) [inline docs/en/interfaces/formats.md:1273]:
```suggestion
To insert data into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) column as an array of structs you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
```

File: docs/en/sql-reference/data-types/nested-data-structures/nested.md [hunk 5]
```diff
@@ -101,4 +103,8 @@ For a DESCRIBE query, the columns in a nested data structure are listed separate
 
 The ALTER query for elements in a nested data structure has limitations.
 
+## Inserting {#inserting}
+
+To insert `Array(Struct)` values into `Nested` columns, you must switch on the [input_format_parquet_import_nested](../../../operations/settings/settings.md#input_format_parquet_import_nested), [input_format_arrow_import_nested](../../../operations/settings/settings.md#input_format_arrow_import_nested) or [input_format_orc_import_nested](../../../operations/settings/settings.md#input_format_orc_import_nested) settings for [Parquet](../../../interfaces/formats.md#data-format-parquet), [Arrow](../../../interfaces/formats.md#data_types-matching-arrow) and [ORC](../../../interfaces/formats.md#data-format-orc) input formats respectively.
+
```
Comment (Avogar) [inline docs/en/sql-reference/data-types/nested-data-structures/nested.md:109]: We should remove this at all, the changes are related only to input formats and we shouldn't write about them here.

File: docs/ru/interfaces/formats.md [hunk 6]
```diff
@@ -1180,14 +1180,16 @@ ClickHouse поддерживает настраиваемую точность
...
-### Вставка и выборка данных {#vstavka-i-vyborka-dannykh}
+### Вставка и выборка данных {#inserting-and-selecting-data}
 
 Чтобы вставить в ClickHouse данные из файла в формате Parquet, выполните команду следующего вида:
 
 ``` bash
 $ cat {filename} | clickhouse-client --query="INSERT INTO {some_table} FORMAT Parquet"
 ```
 
+Чтобы вставить данные типа `Array(Struct)` в колонки типа [Nested](../sql-reference/data-types/nested-data-structures/nested.md), нужно включить настройку [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested).
```
Comment (Avogar) [inline docs/ru/interfaces/formats.md:1191]: The same, let's write this way:
"Чтобы вставлять данные в колонки типа [Nested](../sql-reference/data-types/nested-data-structures/nested.md) в виде массива структур, нужно ..."

File: docs/en/operations/settings/settings.md [hunk 7]
```diff
@@ -260,6 +260,39 @@ If an error occurred while reading rows but the error counter is still less than
 
 If both `input_format_allow_errors_num` and `input_format_allow_errors_ratio` are exceeded, ClickHouse throws an exception.
 
+## input_format_parquet_import_nested {#input_format_parquet_import_nested}
+
+Enables or disables the ability to insert `Array(Struct)` values into [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) columns in [Parquet](../../interfaces/formats.md#data-format-parquet) input format.
```
Comment (Avogar) [inline docs/en/operations/settings/settings.md:265]: Let's change here and below too:

"Enables or disables the ability to insert data into [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) columns as an array of structs in ..."

and 

"0 — Data can not be inserted into `Nested` columns as an array of structs.
-   1 —Data can be inserted into `Nested` columns as an array of structs."

And in Russian version

File: docs/en/operations/settings/settings.md [hunk 8]
```diff
@@ -260,6 +260,39 @@ If an error occurred while reading rows but the error counter is still less than
...
+
+Possible values:
+
+-   0 — `Array(Struct)` values can not be inserted into `Nested` columns.
+-   1 — `Array(Struct)` values can be inserted into `Nested` columns.
+
```
Comment (Avogar) [inline docs/en/operations/settings/settings.md:271]: Let's change here and below too:

`Enables or disables the ability to insert data into [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) columns as an array of structs in ...`

and
 
```
-   0 — Data can not be inserted into `Nested` columns as an array of structs.
-   1 — Data can be inserted into `Nested` columns as an array of structs.
```

And in Russian version

Comment (gyuton) [reply docs/en/operations/settings/settings.md]: Done.

**Review by Avogar — APPROVED** [review]

**Pull request merged**